pages/common/msmtp.md

```diff
@@ -0,0 +1,16 @@
+# msmtp
+
+> An SMTP client.
+> In the default sendmail mode, msmtp reads a mail from standard input and sendscit to an SMTP server for delivery.
+
+- Send a mail using the default account configured in `~/.msmtprc`:
+
+`echo {{"Hello world"}} | msmtp {{to@example.org}}`
+
+- Send a mail using a specific account configured in `~/.msmtprc`:
+
+`echo {{"Hello world"}} | msmtp --account={{example}} {{to@example.org}}`
+
+- Send a mail without a configured account:
+
+`echo {{"Hello world"}} | msmtp --host={{localhost}} --port={{999}} --from={{from@example.org}} {{to@example.org}}`
```

Comment:
How does this associate the email "from@example.org" with the example.org mail servers? I'm assuming the recipient would see something different in the "From" field when they use, say, gmail to read the message than if the actual mail servers at example.com had been used to send the message, no?